yarn.lock

```diff
@@ -2053,6 +2070,11 @@ kind-of@^6.0.3:
   resolved "https://registry.yarnpkg.com/kind-of/-/kind-of-6.0.3.tgz#07c05034a6c349fa06e24fa35aa76db4580ce4dd"
   integrity sha512-dcS1ul+9tmeD95T+x28/ehLgd9mENa3LsvDTtzm3vyBEO7RPptvAD+t44WVXaUjTBRcrpFeFlC8WCruUR456hw==
 
+kleur@^3.0.3:
```

Comment:
Just to point out: chalk could be replaced by kleur cause kleur is a peer dependency of prompts. But the latest version has the major version 4 so it's fine to use chalk.

Another coloring option would be the one from debug https://www.npmjs.com/package/debug#namespace-colors.

No need to update though, chalk is doing the job.